### DISCUSSION TITLE
font does not effect the android app

### DISCUSSION THREAD

**Discussion by arshamX** [AvaloniaUI/Avalonia#15025] [Q&A]
hello every body I have font which I am using in my application I copied my font to Assets/Fonts directory now the font is effected the a desktop app but has no effect on android app is there any one who could help  
I tried every thing I thought could be affective

**Comment by timunie** [comment]
Is the font installed on your desktop machine by any chance?

**Comment by maxkatz6** [comment]
Make sure you followed all steps
https://docs.avaloniaui.net/docs/guides/styles-and-resources/how-to-use-fonts
https://docs.avaloniaui.net/docs/basics/user-interface/assets
